cymbaline/generators.py

```diff
@@ -0,0 +1,63 @@
+from cymbaline.data import DataProvider
+from cymbaline.language import LanguageToolkit
+
+import random
+
+
+class Generator:
+    def __init__(self, context_word=None, context_vector=None):
+    """Initiates Generator with either single word or vector, which defines the context"""
```

Comment:
Incorrect level of indentation for the docstring. 
```suggestion
        """Initiates Generator with either single word or vector, which defines the context"""
```

If you want to improve your docstrings (though this one is perfect), refer to [PEP 257 - Docstring Conventions](https://www.python.org/dev/peps/pep-0257/).

Reply:
Shame on me!